src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,6 +30,17 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
```

Comment:
it doesn't seem like you updated the help? please take note that you need to update the help, tests and user guide when you make such modifications.